src/app/(dashboard)/festivals/[id]/action.ts

```diff
@@ -0,0 +1,24 @@
+"use server";
+
+type RequestType = {
+  subId: string;
+  name: string;
+  festivalSubId: string;
+  startDate: string;
+  endDate: string;
+  location: string;
+  description: string;
+}
+
+export async function createConcert(values: RequestType) {
+  const res = await fetch('/admin/concert', {
+    method: 'POST',
```

Comment:
```suggestion
    method: 'POST',
    credential: 'include'
```
쿠키 전달을 하려면, 서버 액션에서도 credential 옵션을 사용해야 하지 않나요?